src/commands/GetInfoCommand.cpp

```diff
@@ -518,8 +518,8 @@ bool GetInfoCommand::SendClips(const CommandContext &context)
          for (WaveClip * pClip : ptrs) {
             context.StartStruct();
             context.AddItem((double)i, "track");
-            context.AddItem(pClip->GetStartTime(), "start");
-            context.AddItem(pClip->GetEndTime(), "end");
+            context.AddItem(pClip->GetPlayStartTime(), "start");
+            context.AddItem(pClip->GetPlayEndTime(), "end");
```

Comment:
As with Nyquist, so perhaps too for macro programming: give more information about the size of the hidden parts of the wave for sophisticated users.  Only information:  no interface for changing it.